### DISCUSSION TITLE
How do I change the background colour of an expansion header?

### DISCUSSION THREAD

**Discussion by iron3oxide** [zauberzeug/nicegui#2215] [Q&A] [answered]
### Question

I know that it can be changed via the `header-style` prop, yet whatever method to set that prop I use, the background colour remains unchanged.

**Comment by rodja** [comment, marked as answer]
Try this:

```py
    with ui.expansion('Expand!').props('header-class="bg-red text-white"').classes('w-48'):
        ui.label('inside the expansion')
```

**Reply by iron3oxide** [reply]
Thanks a lot! I think when one uses the `bg-primary` and `rounded-borders` classes it looks pretty neat.
